react/README.md

```diff
@@ -18,3 +29,259 @@ For more info, see [Getting started with React](https://facebook.github.io/react
+
+- When the component does more than just render `props`, such as attaching callback handlers, the stateless function can become unnecessarily complex with nested functions
+- [`propTypes`](#component-propTypes) are defined _within_ the ES6 class, but have to be defined as additional properties _outside_ of a stateless function
+- Using ES6 classes for the main/default component help differentiate it from [helper components](#)
+
+## Component organization
+
+Export only one component per file as the default (eslint: [`react/no-multi-comp`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-multi-comp.md))
+
+```jsx
+// good
+export default class MainComponent extends React.Component {
+
+}
+
+
+// bad (exports multiple components)
+export class ComponentA extends React.Component {
+
+}
+
+export class ComponentB extends React.Component {
+
+}
+```
+
+## Component reference naming
+
+Use PascalCase for React components and camelCase for their instances (eslint: [`react/jsx-pascal-case`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-pascal-case.md)).
+
+Components:
+```jsx
+// good
+import TextInput from './atoms/TextInput';
+
+// bad (uses camelCase for component reference)
+import textInput from './atoms/TextInput';
+```
+
+Component instances:
+```jsx
+// good
+let emailField = (<TextInput name="email" />);
+
+// bad (uses PascalCase for component instances)
+let EmailField = (<TextInput name="email" />);
+```
+
+## Component `propTypes`
+
+Coming soon...
+
+## Helper components
+
+Coming soon...
+
+## Component method ordering
+
+Coming soon...
+
+## JSX Wrapping
+
+Always wrap JSX tags in parentheses as a signal that we're transitioning from vanilla JavaScript to JSX (eslint: [`react/wrap-multilines`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/wrap-multilines.md)):
+
+```jsx
+// good (multi-line JSX wrapped in parentheses)
+render() {
+    return (
+        <Component value="foo">
+            <ChildComponent />
+        </Component>
+    );
+}
+
+// good (single-line JSX is also wrapped in parentheses)
+let content = (<div>Content</div>);
+
+// bad (missing wrapping parentheses for multi-line)
+render() {
+    return <Component value="foo">
+        <ChildComponent />
+    </Component>;
+}
+
+// bad (missing wrapping parentheses for single-line)
+let content = <div>Content</div>
+```
+
+## JSX alignment
+
+When a component has four props or less with no content, the tag can be on a single line (eslint: [`react/jsx-max-props-per-line`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-max-props-per-line.md)):
```

Comment:
I feel like 4 might even be too many, could cause inconsistency (hard to always keep that in mind / seems arbitrary). Maybe any more than two should be split to single line to increase similarity/readability between components? Just a thought